tests/test_arraymanipulation.py

```diff
@@ -605,3 +601,10 @@ def test_2D_array2(self):
     def test_generator(self):
         with assert_warns(FutureWarning):
             dpnp.vstack((numpy.arange(3) for _ in range(2)))
+
+
+@pytest.mark.parametrize("arys", [1.0, dpnp.arange(9.0).reshape(3, 3), [3, 4]])
+def test_atleast_1d(arys):
+    expected = numpy.atleast_1d(arys)
```

Comment:
can we add more tests, probably similar to what numpy does in `numpy/core/tests/test_shape_base.py` for `atleast_1d`?